client/blocks/jetpack-benefits/general-benefits.tsx

```diff
@@ -121,7 +106,7 @@ const JetpackGeneralBenefits: React.FC< Props > = ( props ) => {
 	}
 
 	// General benefits of all Jetpack Plans (brute force protection, CDN)
-	if ( hasJetpackPlanSlug ) {
+	if ( planHasFeature( productSlug, FEATURE_STANDARD_SECURITY_TOOLS ) ) {
```

Comment:
The "Brute force attack protection and downtime monitoring." info line isn't showing up on Jetpack Complete plans. I think because `FEATURE_STANDARD_SECURITY_TOOLS` isn't included in the [Jetpack Complete plans features list](https://github.com/Automattic/wp-calypso/blob/a33fdef25222f1052a5fa68b66424911900f4683/packages/calypso-products/src/plans-list.tsx#L988-L1011)? Maybe we should consider using a different feature here?

Reply:
Make sense to me. I feel like no conditional check at all here would be fine.